cluster/engine.go

```diff
@@ -423,6 +423,10 @@ func (e *Engine) updateSpecs() error {
 	}
 	for _, label := range info.Labels {
 		kv := strings.SplitN(label, "=", 2)
+		if len(kv) != 2 {
+			message := fmt.Sprintf("Engine (ID: %s, Addr: %s) has a label (%s) whose format is not like \"key=value\". Please check engine's label setting.", e.ID, e.Addr, label)
```

Comment:
This could be reworded to remove the "like".  ""Engine (ID: %s, Addr: %s) contains an invalid label (%s) not formatted as \"key=value\"."